dspace/config/launcher.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
```

Comment:
Add a comment to launcher.xml that links to the new documentation for adding command line tasks.